tusclient/request.py

```diff
@@ -54,6 +61,13 @@ def status_code(self):
         """
         return self.handle.getinfo(pycurl.RESPONSE_CODE)
 
+    @property
+    def return_value(self):
```

Comment:
maybe we should rename this property to `response_content`, so it is clearer what it is?